app/controllers/sprangular/orders_controller.rb

```diff
@@ -13,9 +13,15 @@ def index
   end
 
   def show
-    authorize! :show, @user
+    token = request.headers['X-Spree-Order-Token']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.